packages/server/src/errors.ts

```diff
@@ -136,7 +68,7 @@ function enrichError(error: Partial<GraphQLError>, debug: boolean = false) {
     delete expanded.extensions.exception;
   }
 
-  return expanded as ApolloError;
+  return expanded as GraphQLError;
 }
 
 export function toApolloError(
```

Comment:
Do we want to remove this altogether? Looks like we only export it.

Reply:
Fixed in updated code.
Now it doesn't have a single typecast 🎉